Python/bytecodes.c

```diff
@@ -657,6 +657,22 @@ dummy_func(
             goto resume_frame;
         }
 
+        register inst(RETURN_VALUE_R, (retval --)) {
```

Comment:
Nice progress!

Reply:
It's much easier to make progress when you stop caring about asyncio.

I'm taking the rest of the year off.